src/main/java/sg/ncl/exceptions/FuturePlanDownloadException.java

```diff
@@ -0,0 +1,11 @@
+package sg.ncl.exceptions;
+
+/**
+ * @author cassie
+ */
+public class FuturePlanDownloadException extends Exception {
+
+    public FuturePlanDownloadException(String message) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)